lib/Service/src/SettingsHandler.cpp

```diff
@@ -160,22 +181,25 @@ bool SettingsHandler::loadConfigurationFile(const String& filename)
 bool SettingsHandler::saveConfigurationFile(const String& filename)
 {
     bool                           isSuccessful = false;
-    const size_t                   maxDocSize   = 1024U;
+    const size_t                   maxDocSize   = 2048U;
     StaticJsonDocument<maxDocSize> doc;

```

Comment:
With 2 kbyte it may make sense to use DynamicJsonDocument.

Reply:
Just checked the size again. It is currently 450 Bytes, so the size can remain at 1024U

Reply:
Same applies for `loadConfigurationFile` method